app/tests/system/setup/setup.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Loop through the custom contexts that exist when the app starts
+ * and grab their names.
+ *
+ * To change the environment we're testing against, specify it in site.json.
+ * This should align with what the app is built against, which is declared in
+ * app/environment.js hostUrl
+ */
+
+var Site = require('../site.json');
+
+var Setup = {};
+
+var selectors = {
+    previewButton: '#authorize',
+    registeredAuctionList: '.js-auction-lists'
+};
+
+var expected = {
+    //EXAMPLE Webviews constructed for project.
+    'noInternet': /no-internet\.html/i,
+    'couldNotLoad': /could-not-load\.html/i
+};
+
+Setup.appPreview = function(browser) {
+    browser.contexts(function(result) {
+        browser
```

Comment:
There are quite a few places where there is a line break after a variable reference that don't seem to be needed.  Here, for instance, there is a single call to `.session()` and the parameter list is short enough that it looks like it could all fit on one line.